vng_api_common/audittrails/viewsets.py

```diff
@@ -2,6 +2,7 @@
 
 from django.db import transaction
 from django.http import Http404
+from drf_spectacular.utils import extend_schema, OpenApiParameter
```

Comment:
De wijzigingen in dit bestand kun je weghalen, er wordt hier qua gedrag niets veranderd